CONTRIBUTING.md

```diff
@@ -28,34 +28,46 @@ yarn start
 
 ## Generators
 
-This repo leverage [Plop](https://plopjs.com/) to help scaffold out the files that you need when you're creating a new component.
+This repo leverages [Plop](https://plopjs.com/) to help scaffold out the files
+that you need when you're creating a new component.
 
 ### New Components
 
-Running `yarn add-component` will create the folder and files you need to start building out your component. Each component at the very least should have an export, documentation, testing, and a story file.
+Running `yarn add-component` will create the folder and files you need to start
+building out your component. Each component at the very least should have an
+export, documentation, testing, and a story file.
 
 ## Releases
 
-TODO: Change this section to reflect publishing on NPM with Changesets.
+Releases happen on every merge to main when a package has changes. Our packages
+are published to NPM using Changesets. For more information about it, click
+[here](https://github.com/changesets/changesets/blob/main/docs/detailed-explanation.md).
 
-1. Create a [Github Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) with packages permissions.
-2. Login into Github Packages (the password should be the token created on the step before)
+### Creating a Changeset
 
-```
-npm login --registry=https://npm.pkg.github.com
+If you are adding a package in a PR, run:
+
+```shell
+yarn changeset
```

Comment:
This should only be run when adding a package? What if you modify an existing one?

Reply:
The contributor needs to run this command when adding or updating something in the packages, so, yes, I will update this message. 🙏🏻